frontend/ui/menusorter.lua

```diff
@@ -133,10 +133,12 @@ function MenuSorter:sort(item_table, order)
     -- they should, however have one going in
     local menu_buttons_offset = 0
     for i,top_menu in ipairs(menu_table["KOMenu:menu_buttons"]) do
-        if menu_table["KOMenu:menu_buttons"][i].sub_item_table then
-            menu_table["KOMenu:menu_buttons"][i-menu_buttons_offset] = menu_table["KOMenu:menu_buttons"][i].sub_item_table
+        local menu_button  = menu_table["KOMenu:menu_buttons"][i].sub_item_table
+        menu_table["KOMenu:menu_buttons"][i] = nil
```

Comment:
It's arbitrary in principle, but within the context of this file this should go at the bottom. It's how all the other cleanup nilling is structured and deviations make it harder to read. :-)

No further comments besides the double space in ` local menu_button  =`

Reply:
>  this should go at the bottom

Yes I understand, but in this case the order is important. It doesn't work if placed after the `if end`

Reply:
Right, you'd have to check `if menu_buttons_offset > 0`. Very well, let's keep it like this.